lib/tokenizer.js

```diff
@@ -5,29 +5,45 @@
 
 /**
  * A function for splitting a string into tokens ready to be inserted into
- * the search index. Uses `lunr.tokenizer.seperator` to split strings, change
+ * the search index. Uses `lunr.tokenizer.separator` to split strings, change
  * the value of this property to change how strings are split into tokens.
  *
  * @module
  * @param {String} obj The string to convert into tokens
- * @see lunr.tokenizer.seperator
+ * @see lunr.tokenizer.separator
  * @returns {Array}
  */
 lunr.tokenizer = function (obj) {
   if (!arguments.length || obj == null || obj == undefined) return []
   if (Array.isArray(obj)) return obj.map(function (t) { return lunr.utils.asString(t).toLowerCase() })
 
-  return obj.toString().trim().toLowerCase().split(lunr.tokenizer.seperator)
+  return obj.toString().trim().toLowerCase().split(lunr.tokenizer.separator)
 }
 
 /**
- * The sperator used to split a string into tokens. Override this property to change the behaviour of
+ * Define a get and set method for seperator [sic], to add legacy support for the misspelling of
+ * separator. Use lunr.tokenizer.separator instead.
+ *
+ * @see lunr.tokenizer.separator
+ * @deprecated since 0.8.0
+ */
+Object.defineProperty(lunr.tokenizer, "seperator", {
```

Comment:
Browser compatibility for defineProperty seems pretty good, but I wasn't sure what the minimum supported versions are in Lunr.

https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#Browser_compatibility